Library/Homebrew/os/mac/ruby_mach.rb

```diff
@@ -21,13 +21,7 @@ def mach_data
       end
 
       machos.each do |m|
-        arch = case m.cputype
-        when "CPU_TYPE_I386" then :i386
-        when "CPU_TYPE_X86_64" then :x86_64
-        when "CPU_TYPE_POWERPC" then :ppc7400
-        when "CPU_TYPE_POWERPC64" then :ppc64
-        else :dunno
-        end
+        arch = m.cputype
```

Comment:
I'd rather we stick with explicitly enumerating all the officially supported architectures instead of accepting everything that `ruby-macho` accepts. This will also avoid the `:ppc` vs. `:ppc7400` mismatch. I was thinking of something like:

```ruby
arch = case m.cputype
when :x86_64, :i386, :ppc64 then m.cputype
when :ppc then :ppc7400
else :dunno
end
```